File: SpaceWar3/SpaceWar3.cs
using Microsoft.Xna.Framework;
using Microsoft.Xna.Framework.Graphics;
using Microsoft.Xna.Framework.Input;

namespace SpaceWar3
{

    public class SpaceWar3 : Game
    {
        private GraphicsDeviceManager graphics;

        public SpaceWar3()
        {
            graphics = new GraphicsDeviceManager(this) { IsFullScreen = true };
            Content.RootDirectory = "Content";
        }

        protected override void Initialize()
        {
            base.Initialize();
        }

        protected override void Update(GameTime gameTime)
        {
            if (GamePad.GetState(PlayerIndex.One).Buttons.Back == ButtonState.Pressed ||
                Keyboard.GetState().IsKeyDown(Keys.Escape))
            {
                Exit();
            }

            base.Update(gameTime);
        }

        protected override void Draw(GameTime gameTime)
        {


            base.Draw(gameTime);
        }
    }
}

File: SpaceWar3/SpaceWar3.fs
namespace DEMW.SpaceWar3

open Microsoft.Xna.Framework
open Microsoft.Xna.Framework.Input;
open System

type SpaceWar3() as this =
    inherit Game()

    let g = new GraphicsDeviceManager(this)

    do
        g.CreateDevice()
        g.ToggleFullScreen()

    let shouldExit = fun () ->
        GamePad.GetState(PlayerIndex.One).Buttons.Back = ButtonState.Pressed ||
        Keyboard.GetState().IsKeyDown(Keys.Escape)

    override this.Initialize() =
        base.Initialize()

    override this.Update(gameTime) =
        if shouldExit() then
            this.Exit()

        base.Update(gameTime);

    override this.Draw(gameTime) =

        g.GraphicsDevice.Clear(Color.CornflowerBlue)

        base.Draw(gameTime)
